pkg/tool/codehub/commit.go

```diff
@@ -0,0 +1,35 @@
+package codehub
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+type CommitList struct {
```

Comment:
it is the full api response data, not a commitList/commit definition. You should define you own  commitList/commit struct.
same as other functions.